### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,17 +15,17 @@
...
-  - Have a daily summary of sales activities 
-  - Leverage an enhanced contact list
-  - View more details about opportunity line items
-  - Gather daily historical records of your accounts, contacts and opportunities
+- Understand the performance of your opportunities
+- Drill into how the members of your sales team are performing
+- Have a daily summary of sales activities
+- Leverage an enhanced contact list
+- View more details about opportunity line items
+- Gather daily historical records of your accounts, contacts and opportunities
```
Comment (fivetran-joemarkiewicz) [inline README.md:28]:
```suggestion
  - Understand the performance of your opportunities
  - Drill into how the members of your sales team are performing
  - Have a daily summary of sales activities
  - Leverage an enhanced contact list
  - View more details about opportunity line items
  - Gather daily historical records of your accounts, contacts and opportunities
```

File: .quickstart/quickstart.yml [hunk 2]
```diff
@@ -23,4 +23,7 @@ destination_configurations:
   databricks:
     dispatch:
       - macro_namespace: dbt_utils
-        search_order: [ 'spark_utils', 'dbt_utils' ]
\ No newline at end of file
+        search_order: [ 'spark_utils', 'dbt_utils' ]
+public_models: [
+
+]
```
Comment (fivetran-joemarkiewicz) [inline .quickstart/quickstart.yml:29]:
```suggestion
public_models: [
  "salesforce__contact_enhanced",
  "salesforce__daily_activity",
  "salesforce__manager_performance",
  "salesforce__opportunity_enhanced",
  "salesforce__opportunity_line_item_enhanced",
  "salesforce__owner_performance",
  "salesforce__sales_snapshot"
]
```

File: .quickstart/quickstart.yml [hunk 3]
```diff
@@ -23,4 +23,13 @@ destination_configurations:
...
+        search_order: [ 'spark_utils', 'dbt_utils' ]
+public_models: [
+  "salesforce__contact_enhanced",
+  "salesforce__daily_activity",
+  "salesforce__manager_performance",
+  "salesforce__opportunity_enhanced",
+  "salesforce__opportunity_line_item_enhanced",
+  "salesforce__owner_performance",
+  "salesforce__sales_snapshot"
+]
```
Comment (fivetran-catfritz) [inline .quickstart/quickstart.yml:35]: @fivetran-joemarkiewicz Do we want to include the [history models](https://github.com/fivetran/dbt_salesforce/tree/main/models/salesforce_history) here?

Comment (fivetran-joemarkiewicz) [reply .quickstart/quickstart.yml:35]: Good question @fivetran-catfritz. The history models are not available in Quickstart. So we will intentionally leave them off the list.

**Review by fivetran-catfritz — APPROVED** [review]
lgtm

**Pull request merged**